xWorkflows/xWorkflow_2/kubecost_query_pod_go/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Start with the official Golang base image
+FROM golang:1.22.4
+
+# Set the Current Working Directory inside the container
+WORKDIR /app
+
+# Copy go mod and sum files
+COPY go.mod go.sum ./
+
+RUN go mod tidy
+
+# Download all dependencies. Dependencies will be cached if the go.mod and go.sum files are not changed
+RUN go mod download
+
+# Copy the source from the current directory to the Working Directory inside the container
+COPY . .
+
+# Build the Go app
+RUN go build -o main .
+
+# Expose port 8080 to the outside world
```

Comment:
Different port in comment and expose command.

Reply:
resolved.